test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamples.cs

```diff
@@ -131,21 +127,20 @@ private void TrainRegression(string trainDataPath, string testDataPath, string m
             // Read the test dataset.
             var testData = reader.Read(new MultiFileSource(testDataPath));
             // Calculate metrics of the model on the test data.
-            // We are using the 'regression' context object here to perform evaluation.
-            var metrics = regression.Evaluate(model.Transform(testData), label: r => r.Target, score: r => r.Prediction);
+            var metrics = mlContext.Regression.Evaluate(model.Transform(testData), label: r => r.Target, score: r => r.Prediction);
 
             using (var stream = File.Create(modelPath))
             {
                 // Saving and loading happens to 'dynamic' models, so the static typing is lost in the process.
-                model.AsDynamic.SaveTo(env, stream);
+                mlContext.Model.Save(model.AsDynamic, stream);
```

Comment:
Thoughts on making this so I don't have to say `model.AsDynamic`?  Can we open an issue to add that overload?

Reply:
absolutely

---
In reply to: [226105313](https://github.com/dotnet/machinelearning/pull/1252#discussion_r226105313) [](ancestors = 226105313)